CriticalMass/Images.xcassets/icon-send.imageset/Contents.json

```diff
@@ -0,0 +1,16 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "icon-send.pdf"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  },
+  "properties" : {
+    "template-rendering-intent" : "template",
+    "preserves-vector-representation" : true
```

Comment:
Cool! Though this is only supported with iOS 11 +

Reply:
Another case for bumping the target up maybe

Reply:
I tested it with the iOS 10 simulator and it seemed to work

Reply:
Thanks for testing